pyfar/plot/line.py

```diff
@@ -351,23 +351,24 @@ def freq_group_delay(signal, log_prefix=20, log_reference=1, ax=None,
 
     with plt.style.context(plotstyle(style)):
         ax = _line._freq_group_delay(signal, log_prefix, log_reference,
-                                     **kwargs)
+                                     ax, **kwargs)
 
     plt.tight_layout()
     Interaction('LineXLog', ax[0], signal, style, **kwargs)
 
     return ax
 
 
-def summary(signal, ax=None, style='light', **kwargs):
+def multi(signal, plots, ax=None, style='light', **kwargs):
     """
-    Plot the time domain, the time domain in dB, the magnitude spectrum,
-    the frequency domain, the phase and group delay.
+    Generate multiple plots of a Signal object.
```

Comment:
The docstring is also not very informational ;)

Reply:
How about `Generate custom subplot layouts of a Signal object.`

Still not the best, but the example at the end can also help to clarify.